src/test/java/org/mule/extension/db/integration/model/AbstractTestDatabase.java

```diff
@@ -104,8 +104,22 @@ public void addAdditionalPlanets(Connection connection) throws SQLException {
     populatePlanetTable(connection, ADDITIONAL_PLANET_VALUES);
   }
 
+  public void removePlanets(Connection connection, Planet[] planets) throws SQLException {
```

Comment:
you could replace `Planet[] planets` with `Planet... planets` so you can create an array directly on the method call `testDatabase.removePlanets(connection, EARTH, MARS, WHATEVER)`